### ISSUE TITLE
[bug] minmer_query failing on sourmash command

### ISSUE THREAD

**Issue by ameynert** [bactopia/bactopia#210]
**Description**

Fresh install of bactopia & build of datasets, testing on a single sample. Fails on minmer_query with unrecognized arguments to sourmash.

```
$ bactopia --sample 20191007_31_pBAD_gyrA --R1 ../raw_data/
20191007/31_pBAD_gyrA/191002_M01270_0094_000000000-CH269_1_11820WB0068L01_1.fastq.gz --R2 ../raw_data/20191007/31_pBAD_gyrA/191002_M01270_0094_000000000-CH269_1_1
1820WB0068L01_2.fastq.gz --datasets ../bactopia_datasets --species "Escherichia coli"
N E X T F L O W  ~  version 20.10.0
Launching `/exports/igmm/eddie/bioinfsvice/ameynert/conda_envs/bactopia/share/bactopia-1.6.x/main.nf` [festering_liskov] - revision: 7a5c0e45eb
bactopia - 1.6.4

Each task will use 4 CPUs out of the available 32 CPUs. At most 8 task(s) will be run at
a time, this can affect the efficiency of Bactopia.

Found 1 Antimicrobial resistance datasets
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/antimicrobial-resistance/amrfinderdb.tar.gz
Found 2 ARIBA datasets
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/ariba/card.tar.gz
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/ariba/vfdb_core.tar.gz
Found 3 PLSDB (plasmid) BLAST files
../bactopia_datasets/plasmid/plsdb.fna.nin
../bactopia_datasets/plasmid/plsdb.fna.nhr
../bactopia_datasets/plasmid/plsdb.fna.nsq
Found 5 minmer sketches/signatures
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/minmer/genbank-k21.json.gz
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/minmer/genbank-k31.json.gz
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/minmer/genbank-k51.json.gz
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/minmer/refseq-k21-s1000.msh
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/plasmid/plsdb.msh
Genome size will be estimated.
Found Prokka proteins file
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/species-specific/escherichia-coli/annotation/proteins.faa
Found Mash Sketch of RefSeq genomes
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/species-specific/escherichia-coli/minmer/refseq-genomes.msh
Found 4 MLST datasets
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/species-specific/escherichia-coli/mlst/achtman/achtman-ariba.tar.gz
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/species-specific/escherichia-coli/mlst/achtman/achtman-blastdb.tar.gz
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/species-specific/escherichia-coli/mlst/pasteur/pasteur-ariba.tar.gz
/gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_datasets/species-specific/escherichia-coli/mlst/pasteur/pasteur-blastdb.tar.gz
If something looks wrong, now's your chance to back out (CTRL+C 3 times).
Sleeping for 5 seconds...
executor >  local (14)
[9e/bcf366] process > gather_fastqs (20191007_31_pBAD_gyrA)         [100%] 1 of 1 ✔
[93/2f8ca2] process > fastq_status (1)                              [100%] 1 of 1 ✔
[f0/626b0e] process > estimate_genome_size (20191007_31_pBAD_gyrA)  [100%] 1 of 1 ✔
[76/4f7283] process > qc_reads (20191007_31_pBAD_gyrA)              [100%] 1 of 1 ✔
[91/f62837] process > qc_original_summary (20191007_31_pBAD_gyrA)   [100%] 1 of 1 ✔
[26/91400e] process > qc_final_summary (20191007_31_pBAD_gyrA)      [100%] 1 of 1 ✔
[11/fa4068] process > assemble_genome (20191007_31_pBAD_gyrA)       [  0%] 0 of 1
[-        ] process > assembly_qc                                   -
[-        ] process > make_blastdb                                  -
[-        ] process > annotate_genome                               -
[67/cdf147] process > count_31mers (20191007_31_pBAD_gyrA)          [100%] 1 of 1 ✔
[-        ] process > sequence_type                                 -
[8a/ca6f3f] process > ariba_analysis (20191007_31_pBAD_gyrA - card) [  0%] 0 of 2
[39/aae678] process > minmer_sketch (20191007_31_pBAD_gyrA)         [ 75%] 3 of 4, failed: 3, retries: 3 
[-        ] process > minmer_query                                  -
[-        ] process > call_variants                                 -
[-        ] process > download_references                           -
[-        ] process > call_variants_auto                            -
[-        ] process > antimicrobial_resistance                      -
[-        ] process > plasmid_blast                                 -
[-        ] process > blast_genes                                   -
[-        ] process > blast_primers                                 -
[-        ] process > blast_proteins                                -
[-        ] process > mapping_query                                 -
[49/29c084] NOTE: Process `minmer_sketch (20191007_31_pBAD_gyrA)` terminated with an error exit status (2) -- Execution is retried (2)
[f1/d116ca] NOTE: Process `minmer_sketch (20191007_31_pBAD_gyrA)` terminated with an error exit status (2) -- Execution is retried (3)

Error executing process > 'minmer_sketch (20191007_31_pBAD_gyrA)'

Caused by:
  Process `minmer_sketch (20191007_31_pBAD_gyrA)` terminated with an error exit status (2)

Command executed [/exports/igmm/eddie/bioinfsvice/ameynert/conda_envs/bactopia/share/bactopia-1.6.x/templates/minmer_sketch.sh]:

  #!/bin/bash
  set -e
  set -u
  LOG_DIR="minmer_sketch"
  mkdir -p ${LOG_DIR}
  echo "# Timestamp" > ${LOG_DIR}/minmer_sketch.versions
  date --iso-8601=seconds >> ${LOG_DIR}/minmer_sketch.versions
  echo "# Mash Version" >> ${LOG_DIR}/minmer_sketch.versions
  mash --version >> ${LOG_DIR}/minmer_sketch.versions 2>&1
  
  echo "# Sourmash Version" >> ${LOG_DIR}/minmer_sketch.versions
  sourmash --version >> ${LOG_DIR}/minmer_sketch.versions 2>&1
  
  # Verify AWS files were staged
  if [[ ! -L "20191007_31_pBAD_gyrA_R1.fastq.gz" ]]; then
      if [ "false" == "true" ]; then
          check-staging.py --fq1 20191007_31_pBAD_gyrA_R1.fastq.gz --is_single
      else
          check-staging.py --fq1 20191007_31_pBAD_gyrA_R1.fastq.gz --fq2 20191007_31_pBAD_gyrA_R2.fastq.gz
      fi
  fi
  
  gzip -cd 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz | mash sketch -o 20191007_31_pBAD_gyrA-k21 -k 21 -s 10000 -r -I 20191007_31_pBAD_gy
rA -
  gzip -cd 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz | mash sketch -o 20191007_31_pBAD_gyrA-k31 -k 31 -s 10000 -r -I 20191007_31_pBAD_gy
rA -
  sourmash compute --scaled 10000 -o 20191007_31_pBAD_gyrA.sig -p 1 \
                      --track-abundance --merge 20191007_31_pBAD_gyrA -k 21,31,51 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz
  
  # pass the FASTQs along
executor >  local (14)
[9e/bcf366] process > gather_fastqs (20191007_31_pBAD_gyrA)         [100%] 1 of 1 ✔
[93/2f8ca2] process > fastq_status (1)                              [100%] 1 of 1 ✔
[f0/626b0e] process > estimate_genome_size (20191007_31_pBAD_gyrA)  [100%] 1 of 1 ✔
[76/4f7283] process > qc_reads (20191007_31_pBAD_gyrA)              [100%] 1 of 1 ✔
[91/f62837] process > qc_original_summary (20191007_31_pBAD_gyrA)   [100%] 1 of 1 ✔
[26/91400e] process > qc_final_summary (20191007_31_pBAD_gyrA)      [100%] 1 of 1 ✔
[11/fa4068] process > assemble_genome (20191007_31_pBAD_gyrA)       [  0%] 0 of 1
[-        ] process > assembly_qc                                   -
[-        ] process > make_blastdb                                  -
[-        ] process > annotate_genome                               -
[67/cdf147] process > count_31mers (20191007_31_pBAD_gyrA)          [100%] 1 of 1 ✔
[-        ] process > sequence_type                                 -
[8a/ca6f3f] process > ariba_analysis (20191007_31_pBAD_gyrA - card) [  0%] 0 of 2
[39/aae678] process > minmer_sketch (20191007_31_pBAD_gyrA)         [100%] 4 of 4, failed: 4, retries: 3 ✘
[-        ] process > minmer_query                                  -
[-        ] process > call_variants                                 -
[-        ] process > download_references                           -
[-        ] process > call_variants_auto                            -
[-        ] process > antimicrobial_resistance                      -
[-        ] process > plasmid_blast                                 -
[-        ] process > blast_genes                                   -
[-        ] process > blast_primers                                 -
[-        ] process > blast_proteins                                -
[-        ] process > mapping_query                                 -
[49/29c084] NOTE: Process `minmer_sketch (20191007_31_pBAD_gyrA)` terminated with an error exit status (2) -- Execution is retried (2)
[f1/d116ca] NOTE: Process `minmer_sketch (20191007_31_pBAD_gyrA)` terminated with an error exit status (2) -- Execution is retried (3)
Error executing process > 'minmer_sketch (20191007_31_pBAD_gyrA)'

Caused by:
  Process `minmer_sketch (20191007_31_pBAD_gyrA)` terminated with an error exit status (2)

Command executed [/exports/igmm/eddie/bioinfsvice/ameynert/conda_envs/bactopia/share/bactopia-1.6.x/templates/minmer_sketch.sh]:

  #!/bin/bash
  set -e
  set -u
  LOG_DIR="minmer_sketch"
  mkdir -p ${LOG_DIR}
  echo "# Timestamp" > ${LOG_DIR}/minmer_sketch.versions
  date --iso-8601=seconds >> ${LOG_DIR}/minmer_sketch.versions
  echo "# Mash Version" >> ${LOG_DIR}/minmer_sketch.versions
  mash --version >> ${LOG_DIR}/minmer_sketch.versions 2>&1
  
  echo "# Sourmash Version" >> ${LOG_DIR}/minmer_sketch.versions
  sourmash --version >> ${LOG_DIR}/minmer_sketch.versions 2>&1
  
  # Verify AWS files were staged
  if [[ ! -L "20191007_31_pBAD_gyrA_R1.fastq.gz" ]]; then
      if [ "false" == "true" ]; then
          check-staging.py --fq1 20191007_31_pBAD_gyrA_R1.fastq.gz --is_single
      else
          check-staging.py --fq1 20191007_31_pBAD_gyrA_R1.fastq.gz --fq2 20191007_31_pBAD_gyrA_R2.fastq.gz
      fi
  fi
  
  gzip -cd 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz | mash sketch -o 20191007_31_pBAD_gyrA-k21 -k 21 -s 10000 -r -I 20191007_31_pBAD_gy
rA -
  gzip -cd 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz | mash sketch -o 20191007_31_pBAD_gyrA-k31 -k 31 -s 10000 -r -I 20191007_31_pBAD_gy
rA -
  sourmash compute --scaled 10000 -o 20191007_31_pBAD_gyrA.sig -p 1 \
                      --track-abundance --merge 20191007_31_pBAD_gyrA -k 21,31,51 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz
  
  # pass the FASTQs along
  mkdir -p fastqs
  if [[ -L "20191007_31_pBAD_gyrA_R1.fastq.gz" ]]; then
      if [ "false" == "false" ]; then
          # Paired-End Reads
          ln -s `readlink 20191007_31_pBAD_gyrA_R1.fastq.gz` fastqs/20191007_31_pBAD_gyrA_R1.fastq.gz
          ln -s `readlink 20191007_31_pBAD_gyrA_R2.fastq.gz` fastqs/20191007_31_pBAD_gyrA_R2.fastq.gz
      else
          # Single-End Reads
          ln -s `readlink 20191007_31_pBAD_gyrA_R1.fastq.gz` fastqs/20191007_31_pBAD_gyrA.fastq.gz
      fi
  else
      if [ "false" == "false" ]; then
          # Paired-End Reads
          cp 20191007_31_pBAD_gyrA_R1.fastq.gz fastqs/20191007_31_pBAD_gyrA_R1.fastq.gz
          cp 20191007_31_pBAD_gyrA_R2.fastq.gz fastqs/20191007_31_pBAD_gyrA_R2.fastq.gz
      else
          # Single-End Reads
          cp  20191007_31_pBAD_gyrA_R1.fastq.gz fastqs/20191007_31_pBAD_gyrA.fastq.gz
      fi
  fi
  
  if [ "false" == "false" ]; then 
      cp .command.err ${LOG_DIR}/minmer_sketch.err
      cp .command.out ${LOG_DIR}/minmer_sketch.out
      cp .command.sh ${LOG_DIR}/minmer_sketch.sh || :
      cp .command.trace ${LOG_DIR}/minmer_sketch.trace || :
  else
      rm -rf ${LOG_DIR}/
  fi

Command exit status:
  2

Command output:
  (empty)

Command error:
  Estimated genome size: 1.0515e+07
  Estimated coverage:    17.1768
  Writing to 20191007_31_pBAD_gyrA-k21.msh...
  Estimated genome size: 1.04713e+07
  Estimated coverage:    15.6953
  Writing to 20191007_31_pBAD_gyrA-k31.msh...
  sourmash: error: unrecognized arguments: -p 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz

Work dir:
  /exports/igmm/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_test/work/bactopia/39/aae67803293e452614135c09c487ca

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line



    Bactopia Execution Summary
    ---------------------------
    Command Line    : nextflow run /exports/igmm/eddie/bioinfsvice/ameynert/conda_envs/bactopia//share/bactopia-1.6.x/main.nf -w /exports/igmm/eddie/bioinfsvice/a
meynert/waclaw_ecoli_evolution/bactopia_test/work/bactopia --sample 20191007_31_pBAD_gyrA --R1 ../raw_data/20191007/31_pBAD_gyrA/191002_M01270_0094_000000000-CH26
9_1_11820WB0068L01_1.fastq.gz --R2 ../raw_data/20191007/31_pBAD_gyrA/191002_M01270_0094_000000000-CH269_1_11820WB0068L01_2.fastq.gz --datasets ../bactopia_dataset
s --species 'Escherichia coli'
    Resumed         : false

    Completed At    : 2021-03-30T19:28:47.977482+01:00
    Duration        : 2m 16s
    Success         : false
    Exit Code       : 2
    Error Report    : Error executing process > 'minmer_sketch (20191007_31_pBAD_gyrA)'

Caused by:
  Process `minmer_sketch (20191007_31_pBAD_gyrA)` terminated with an error exit status (2)

Command executed [/exports/igmm/eddie/bioinfsvice/ameynert/conda_envs/bactopia/share/bactopia-1.6.x/templates/minmer_sketch.sh]:

  #!/bin/bash
  set -e
  set -u
  LOG_DIR="minmer_sketch"
  mkdir -p ${LOG_DIR}
  echo "# Timestamp" > ${LOG_DIR}/minmer_sketch.versions
  date --iso-8601=seconds >> ${LOG_DIR}/minmer_sketch.versions
  echo "# Mash Version" >> ${LOG_DIR}/minmer_sketch.versions
  mash --version >> ${LOG_DIR}/minmer_sketch.versions 2>&1

  echo "# Sourmash Version" >> ${LOG_DIR}/minmer_sketch.versions
  sourmash --version >> ${LOG_DIR}/minmer_sketch.versions 2>&1

  # Verify AWS files were staged
  if [[ ! -L "20191007_31_pBAD_gyrA_R1.fastq.gz" ]]; then
      if [ "false" == "true" ]; then
          check-staging.py --fq1 20191007_31_pBAD_gyrA_R1.fastq.gz --is_single
      else
          check-staging.py --fq1 20191007_31_pBAD_gyrA_R1.fastq.gz --fq2 20191007_31_pBAD_gyrA_R2.fastq.gz
      fi
  fi

  gzip -cd 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz | mash sketch -o 20191007_31_pBAD_gyrA-k21 -k 21 -s 10000 -r -I 20191007_31_pBAD_gy
rA -
  gzip -cd 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz | mash sketch -o 20191007_31_pBAD_gyrA-k31 -k 31 -s 10000 -r -I 20191007_31_pBAD_gy
rA -
  sourmash compute --scaled 10000 -o 20191007_31_pBAD_gyrA.sig -p 1 \
                      --track-abundance --merge 20191007_31_pBAD_gyrA -k 21,31,51 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz

  # pass the FASTQs along
  mkdir -p fastqs
  if [[ -L "20191007_31_pBAD_gyrA_R1.fastq.gz" ]]; then
      if [ "false" == "false" ]; then
          # Paired-End Reads
          ln -s `readlink 20191007_31_pBAD_gyrA_R1.fastq.gz` fastqs/20191007_31_pBAD_gyrA_R1.fastq.gz
          ln -s `readlink 20191007_31_pBAD_gyrA_R2.fastq.gz` fastqs/20191007_31_pBAD_gyrA_R2.fastq.gz
      else
          # Single-End Reads
          ln -s `readlink 20191007_31_pBAD_gyrA_R1.fastq.gz` fastqs/20191007_31_pBAD_gyrA.fastq.gz
      fi
  else
      if [ "false" == "false" ]; then
          # Paired-End Reads
          cp 20191007_31_pBAD_gyrA_R1.fastq.gz fastqs/20191007_31_pBAD_gyrA_R1.fastq.gz
          cp 20191007_31_pBAD_gyrA_R2.fastq.gz fastqs/20191007_31_pBAD_gyrA_R2.fastq.gz
      else
          # Single-End Reads
          cp  20191007_31_pBAD_gyrA_R1.fastq.gz fastqs/20191007_31_pBAD_gyrA.fastq.gz
      fi
  fi

  if [ "false" == "false" ]; then
      cp .command.err ${LOG_DIR}/minmer_sketch.err
      cp .command.out ${LOG_DIR}/minmer_sketch.out
      cp .command.sh ${LOG_DIR}/minmer_sketch.sh || :
      cp .command.trace ${LOG_DIR}/minmer_sketch.trace || :
  else
      rm -rf ${LOG_DIR}/
  fi

Command exit status:
  2

Command output:
  (empty)

Command error:
  Estimated genome size: 1.0515e+07
  Estimated coverage:    17.1768
  Writing to 20191007_31_pBAD_gyrA-k21.msh...
  Estimated genome size: 1.04713e+07
  Estimated coverage:    15.6953
  Writing to 20191007_31_pBAD_gyrA-k31.msh...
  sourmash: error: unrecognized arguments: -p 20191007_31_pBAD_gyrA_R1.fastq.gz 20191007_31_pBAD_gyrA_R2.fastq.gz

Work dir:
  /exports/igmm/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_test/work/bactopia/39/aae67803293e452614135c09c487ca

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line

    Launch Dir      : /gpfs/igmmfs01/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_test
    Working Dir     : /exports/igmm/eddie/bioinfsvice/ameynert/waclaw_ecoli_evolution/bactopia_test/work/bactopia (Total Size: 8.577G)
    Working Dir Size: 8.577G


WARN: Killing pending tasks (3)

```

**Steps to Reproduce**

```
conda create -n bactopia -c conda-forge -c bioconda bactopia
conda activate bactopia
bactopia datasets --species 'Escherichia coli' --outdir bactopia_datasets --include_genus
mkdir bactopia_test
cd bactopia_test

bactopia --sample 20191007_31_pBAD_gyrA \
  --R1 ../raw_data/20191007/31_pBAD_gyrA/191002_M01270_0094_000000000-CH269_1_11820WB0068L01_1.fastq.gz \
  --R2 ../raw_data/20191007/31_pBAD_gyrA/191002_M01270_0094_000000000-CH269_1_11820WB0068L01_2.fastq.gz \
  --datasets ../bactopia_datasets --species "Escherichia coli"
```

**Expected Behavior**
A description of what is expected to happen.

**Execution Environment**

- Bactopia version 1.6.4
- OS Linux, local executor
- Conda

**Additional Information**

I got fairly far with updating the datasets as in #208 then ran into the same error as in #205, so tried a fresh installation and dataset build.

**Comment by ameynert** [comment]
This is fixed by manual editing in of the changes here: https://github.com/bactopia/bactopia/commit/6a58f8f5356f0ac680aece5eb98c578841a35eae

**Comment by ameynert** [comment]
Removing line 42 `--coverage !{params.prokka_coverage} \` from annotate_genome.sh has got me farther. Now Prokka wants blastp at least version 2.2, even though it's already got 2.11. It's a bug in the tool version checking in Prokka 1.13.

```
  [20:14:06] Determined blastp version is 2.11
  [20:14:06] Prokka needs blastp 2.2 or higher. Please upgrade and try again.
```

Upgrading Prokka as suggested: https://github.com/tseemann/prokka/issues/544 - trying a straight `conda install -c bioconda prokka=1.14.6` failed, conda couldn't solve it. Instead, trashed the old annotate_genome-1.6.x environment, updated annotate_genome.yml from latest on github, and re-create the environment in full.

```
conda env create -n annotate_genome-1.6.x -f ../linux/annotate_genome.yml
```

This worked, had a successful run.

**Comment by rpetit3** [comment]
Hey that's great news! 

I've fixed the sourmash 4.0.0 issues here https://github.com/bactopia/bactopia/commit/6a58f8f5356f0ac680aece5eb98c578841a35eae as well as the Prokka 1.14.6 here https://github.com/bactopia/bactopia/commit/47668d4b720ca082e1572a49896d0345c6b601d5

Haha you caught Bactopia in a turbulent time dealing with changes in the conda envs. To help deal with this in the future I've pinned the program versions in the update script I have (https://github.com/bactopia/bactopia/blob/master/bin/update-conda.sh#L51) to hopefully prevent the sometimes weird choices conda makes.

I'm going to push a version update tonight or tomorrow.
